src/contracts/middleware/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -344,8 +388,15 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
     }
     
     function _setChurnApprover(address newChurnApprover) internal {
+        address prevChurnApprover = churnApprover;
         churnApprover = newChurnApprover;
-        emit ChurnApproverUpdated(newChurnApprover);
+        emit ChurnApproverUpdated(prevChurnApprover, newChurnApprover);
```

Comment:
typically you just emit the event first rather than storing things in memory.
so it would look like:
```
emit ChurnApproverUpdated(churnApprover, newChurnApprover);
churnApprover = newChurnApprover;
```

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/142/commits/7eef90501cca5c18d0bd82e76d092013992e4f31 events after ftw, but ye u right